### ISSUE TITLE
Missing 4.2ga9 tag

### ISSUE THREAD

**Issue by ivandov** [pmattes/x3270#64]
The [x3270 wiki download page](https://x3270.miraheze.org/wiki/Downloads) references the latest stable build as `4.2ga9`, however the tags on this repository currently only have `4.2ga8` available. 

Is `4.2ga9` considered unstable? Or is it just that the tag is missing? 

Could a new tag please be created? Looks like it would need to point to this commit id: https://github.com/pmattes/x3270/tree/1627b3dcf40216fc6ac960394be3ac60ee1773f4

**Comment by pmattes** [comment]
Yes, this was a simple oversight on my part. I've pushed the new tag to github and Sourceforge.
